packages/react/src/toast/Toast.tsx

```diff
@@ -18,6 +18,7 @@ type ToastProps = BoxProps<
 
 const iconMap = {
   danger: IconCircleExclamationFilled,
+  informational: IconCircleInfoFilled,
```

Comment:
this should be information

```suggestion
  information: IconCircleInfoFilled,
```